style.css

```diff
@@ -116,4 +118,836 @@
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card6_title6 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card6_paragraph {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    width: 85%;
+    color: #fff;
+    line-height: 24px;
+    margin-bottom: 20px;
+  }
+
+  .buttons6 {
+    display: flex;
+    width: 85%;
+    justify-content: flex-start;
+    list-style: none;
+    padding-left: 0;
+    gap: 10px;
+  }
+
+  .li_buttons6 {
+    background: rgba(255, 255, 255, 0.24);
+    color: #fff;
+    padding: 10px 12px;
+    width: auto;
+    height: 40px;
+    border: none;
+    align-items: center;
+  }
+
+  .card6button {
+    margin-top: 20%;
+    width: 100%;
+    height: 48px;
+    background: #ff6b00;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 17px;
+    padding: 12px;
+    border: none;
+  }
+
+  /* Card 7 */
+
+  .card7 {
+    display: flex;
+    grid-row-start: 7;
+    grid-row-end: 8;
+    flex-direction: column;
+    align-items: center;
+    margin-left: 5%;
+    margin-right: 5%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
+    background-image: url(./Images/Professional\ art\ printing.svg);
+    height: 500px;
+  }
+
+  .card7_title {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    margin-top: 40%;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card7_title7 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card7_paragraph {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    width: 85%;
+    color: #fff;
+    line-height: 24px;
+    margin-bottom: 20px;
+  }
+
+  .buttons7 {
+    display: flex;
+    width: 85%;
+    justify-content: flex-start;
+    list-style: none;
+    padding-left: 0;
+    gap: 10px;
+  }
+
+  .li_buttons7 {
+    background: rgba(255, 255, 255, 0.24);
+    color: #fff;
+    padding: 10px 12px;
+    width: auto;
+    height: 40px;
+    border: none;
+    align-items: center;
+  }
+
+  .card7button {
+    margin-top: 20%;
+    width: 100%;
+    height: 48px;
+    background: #ff6b00;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 17px;
+    padding: 12px;
+    border: none;
+  }
+
+  /* About Section */
+
+  .aboutme_section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    background-color: #1c1a19;
+    background-image: url(./Images/illustration-about-me-mobile.svg);
+    background-repeat: no-repeat;
+    background-position: 100% 23%;
+  }
```

Comment:
- [ ] As I said above ⬆️ please, we must stick to the design as much as possible.
- Your design:
![image](https://user-images.githubusercontent.com/58642949/185516941-ed5d14ca-a222-45b6-bc35-a252a3db38e4.png)
- Figma's template:
![image](https://user-images.githubusercontent.com/58642949/185516984-f3177f71-69e0-4c0d-a0b1-d268a93ac574.png)